packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -240,6 +247,22 @@ export const GiantTrackTile = ({
   const showPlay = isUSDCPurchaseGated ? hasStreamAccess : true
   const shouldShowScheduledRelease =
     isScheduledRelease && dayjs(releaseDate).isAfter(dayjs())
+  const [isDescriptionExpanded, setIsDescriptionExpanded] = useState(false)
+  const [showToggle, setShowToggle] = useState(false)
+  const descriptionRef = useRef<HTMLDivElement>(null)
+
+  // Calculate line height and determine if toggle should be shown
+  useEffect(() => {
```

Comment:
I know we have something on the profile left nav for this right? Mind checking implementation there? I wonder if this is something we could build into user generated text?

Reply:
oh wow it's way better over there